clrsvsim/simulator.py

```diff
@@ -211,13 +212,13 @@ def modify_copy_number(input_bam, output_bam, chrom, start, end, ratio_change, s
     Returns:
         A tuple of: (number of reads in the region, number of reads written in the region, number of split reads written)
     """
-
-    # TODO: replace random seq with actual seq taken from location of duplication/deletion.
-    random_seed = int(random_seed) if random_seed else 2016
-    random.seed(random_seed)
-    random_seq = [random_base() for _ in range(1000)]
-    alternate_seq = ''.join(random_seq)
-    reversed_alternate_seq = ''.join(reversed(random_seq))
+    fa = pyfasta.Fasta(ref_genome)
```

Comment:
I don't think so because there aren't too many of these in non-weird portions of the genome.